test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
```

Comment:
@soltysh is there any reason that an OnBuild strategy might not have the assemble script available to it when trying to build?  http://pastebin.test.redhat.com/295467